healthtools/scrapers/base_scraper.py

```diff
@@ -225,43 +229,41 @@ def archive_data(self, payload):
         '''
         Upload scraped data to AWS S3
         '''
-        data_key = DATA_DIR + self.data_key
-        data_archive_key = DATA_DIR + self.data_archive_key
```

Comment:
@Gathondu Yip. This would be correct. You have to surround the get object function [here](https://github.com/Gathondu/healthtools_ke/blob/d7020836203393bd8b716ce1f8c9d15e1cb20109/healthtools/scrapers/base_scraper.py#L235) with a try-catch.

If it doesn't exist, you create the new files
else if they do exist, you check if it matches the latest.json.
If latest.json is same, it goes [here](https://github.com/Gathondu/healthtools_ke/blob/d7020836203393bd8b716ce1f8c9d15e1cb20109/healthtools/scrapers/base_scraper.py#L249), else it goes [here](https://github.com/Gathondu/healthtools_ke/blob/d7020836203393bd8b716ce1f8c9d15e1cb20109/healthtools/scrapers/base_scraper.py#L238).